### PR TITLE
Link to Quarkus docs extension

### PR DESCRIPTION
**Pull request by holly-cummins**
I noticed it's hard to find the quarkus extension docs page, and our instructions for writing new extensions don't seem to mention it. The quarkiverse wiki seems like the right place, so I've added a reference there.

### CODE REVIEW

File: docs/Home.md [hunk 1]
```diff
@@ -56,38 +56,43 @@ mvn io.quarkus:quarkus-maven-plugin:create-extension \
...
 
 All Quarkiverse projects should have a build and test suite that can be parameterized to use specific Quarkus and Platform versions to ensure we can get
 early warning of issues of master builds; but also to verify compatibiltiy with maintanence branch/tags.
 
 Mechanism is To be defined, but something like having a script that will take quarkus core and quarkus platform Group, Artifact and IDs.
 
+## Documenting your extension
+
+Consider adding documentation to the [Quarkiverse docs page](https://quarkiverse.github.io/quarkiverse-docs/index/index/index.html). 
+You can submit changes via github](https://github.com/quarkiverse/quarkiverse-docs).
```
Comment (gsmet) [inline docs/Home.md]:
```suggestion
You can submit changes via [GitHub](https://github.com/quarkiverse/quarkiverse-docs).
```

**Review by gsmet — CHANGES_REQUESTED** [review]

File: docs/Home.md (hunk 1 above)
Comment (gsmet) [reply docs/Home.md]: (the link absolutely needs fixing, the case is less important but while you're at it...)

Comment (holly-cummins) [reply docs/Home.md]: I'm staring at it and staring at it, and I can't see a difference in the before and after on the links. I'll fix the case on GitHub, but want to make sure an important correction doesn't get missed.

Comment (gsmet) [reply docs/Home.md]: There is a missing `[` that I added in my suggestion.

Comment (holly-cummins) [reply docs/Home.md]: Ohhh I see it. I was looking in the wrong place. Thanks!

**Review by gastaldi — APPROVED** [review]

**Pull request merged**